tests/test_hssm.py

```diff
@@ -217,3 +217,18 @@ def test_model_with_approx_differentiable_likelihood_type(data_angle):
     assert model.loglik == download_hf(loglik)
     trace = model.sample()
     assert isinstance(trace, az.InferenceData)
+
+
+def test_sample_prior_predictive(data):
```

Comment:
Prob wise to add a test that involves multiple regressions, and one that involves hierarchical terms. 
just to make sure nothing funny happens. 

Ok if you disagree, but if so, please add a comment in the code that explains why these tests cover abstractly speaking all cases of interest.

Reply:
Added tests